### DISCUSSION TITLE
GiD Problem Type Questions

### DISCUSSION THREAD

**Discussion by murphy-john** [Anura3D/Anura3D_OpenSource#164] [Q&A] [answered]
@gdicarluccio I have a couple questions about the GiD interface. 
1) Is there a way to set or change the default value for the "Opacity" of a group? When I create a group, the opacity is set to the maximum of 255 (meaning the layer doesn't show up in the render screen). I have to manually change the Opacity of the layer for every layer. 

2) For the material point specification with the double point method, is there a way to set the number of liquid or solid MPs back to zero after a number of MPs has been selected? I'm having trouble where I select the number of solid or liquid material points but can't set them back to zero. Is there a way to delete this specification or add zero to the dropdown menu? My current work around is having to delete and recreate the group which is quite clumsy. EDIT: It looks like GiD is automatically filling the solid and/or liquid MP specification after I've closed the "group". Is there a way to fix this as well?

<img width="358" alt="Screenshot 2023-11-30 104053" src="https://github.com/Anura3D/Anura3D_OpenSource/assets/47335381/1b8b0312-1d12-432c-bffd-f1457a755198">
<img width="518" alt="Screenshot 2023-11-30 104942" src="https://github.com/Anura3D/Anura3D_OpenSource/assets/47335381/8b41e9ae-e11f-42c0-8ccd-7b1cd51e722e">

**Comment by gdicarluccio** [comment, marked as answer]
Hi John, I just added to the interface the option "0"
